explainability-core/src/test/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMDTest.java

```diff
@@ -138,4 +182,55 @@ void testProductionData() {
             e.printStackTrace();
```

Comment:
remove the `e.printStackTrace()`/exception catching: the test case will throw + stacktrace any error it finds, that's the _point_ of a test

Reply:
fixed